src/src/Helpers/Networking.cpp

```diff
@@ -1544,7 +1544,31 @@ int http_authenticate(const String& logIdentifier,
     event += '=';
     event += httpCode;
     eventQueue.addMove(std::move(event));
+    #if FEATURE_THINGSPEAK_EVENT
```

Comment:
A define for this should also be added to `src/src/CustomBuild/define_plugin_sets.h`

```c++
#ifndef FEATURE_THINGSPEAK_EVENT
#ifdef LIMIT_BUILD_SIZE
#define FEATURE_THINGSPEAK_EVENT 0
#else
#define FEATURE_THINGSPEAK_EVENT 1
#endif

#endif
```

Reply:
added it but not sure if it is in the right place.... i was kind of lost in this file :)

Reply:
At the end of that file, you need to define those features when they aren't already set (e.g. in a Custom.h file)

Reply:
ok i´ll change that

Reply:
btw. it was my intention to only make this available in custom builds to not uses any extra space in normal builds since it is not a feature a lot of people will use... but if you think it is ok then we´ll keep it like that. :)